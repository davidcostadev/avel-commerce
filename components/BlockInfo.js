import React from 'react'
import PropTypes from 'prop-types'
import styles from '../assets/scss/App.scss'

const contentEscape = content => ({
  __html: content,
})

const BlockInfo = ({ content }) => (
  <section className={styles.blockInfo}>
    <header className={styles.blockInfoHeader}>
      <h2>Informações do Produto</h2>
    </header>
    <div className={styles.blockInfoContent} dangerouslySetInnerHTML={contentEscape(content)} />
  </section>
)

BlockInfo.propTypes = {
  content: PropTypes.string.isRequired,
}

export default BlockInfo
