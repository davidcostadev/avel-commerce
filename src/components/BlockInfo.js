import React from 'react';
// import classNames from 'classnames';

import styles from '../assets/scss/App.scss';

const BlockInfo = () => (
  <section className={styles.blockInfo}>
    <header className={styles.blockInfoHeader}>
      <h2>Informações do Produto</h2>
    </header>
    <div className={styles.blockInfoContent}>
      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos veniam id, sunt dicta reiciendis</p>
      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos veniam id, sunt dicta reiciendis</p>
      <p>Lorem ipsum dolor sit amet, consectetur adipisicing elit. Dignissimos veniam id, sunt dicta reiciendis</p>
    </div>
  </section>
);

export default BlockInfo;
