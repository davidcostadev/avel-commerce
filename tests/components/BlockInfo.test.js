/* eslint no-undef: "off" */
import React from 'react'
import renderer from 'react-test-renderer'
import BlockInfo from '../../components/BlockInfo'

const content = `
  <strong>strong</strong>
`

describe('Component <BlockInfo />', () => {
  it('<BlockInfo />', () => {
    const tree = renderer.create((
      <BlockInfo
        content={content}
      />
    )).toJSON()
    expect(tree).toMatchSnapshot()
  })
})
