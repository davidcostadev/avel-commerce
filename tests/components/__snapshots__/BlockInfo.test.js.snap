// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Component <BlockInfo /> <BlockInfo /> 1`] = `
<section
  className="App__blockInfo___292Je"
>
  <header
    className="App__blockInfoHeader___1TAq-"
  >
    <h2>
      Informações do Produto
    </h2>
  </header>
  <div
    className="App__blockInfoContent___3WDCW"
    dangerouslySetInnerHTML={
      Object {
        "__html": "
  <strong>strong</strong>
",
      }
    }
  />
</section>
`;
